### DISCUSSION TITLE
Daily 29/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#474] [Daily Sprint]
Con respecto al proyecto:

¿Qué hice ayer?
¿Qué haré hoy?
Impedimentos o dificultades

**Comment by Brian-Almada** [comment]
Este fin de semana me dediqué a estudiar. Me repartí entre profundizar con los property binding y data binding en general y pude entender más de backend. Ya pude hacer mi super usuario, entrar en el servidor y ver todas las cosas que hay ahí, leer el repositorio y tratar de entender para que sirven las cosas, tratar de entender las diferentes carpetas, en fin. Hoy voy a tratar de hacer algunas issues, espero poder hacer alguna ya que cada vez que avanzamos veo que me cuesta aplicar lo que aprendo con las tareas que debo hacer. Creo que soy muy lento para la lógica. Así que espero sortear esa dificultad hoy.

**Comment by rpgrca** [comment]
Ayer hice la registración, lo último que me quedaba era que grabe el password bien en la base de datos que pude hacerlo a la noche. Hoy estuve en la oficina así que no hice mucho, sin embargo me gustaría rearmar un poco lo de la registración porque por ahora lo estoy procesando directamente en el post pero creo que todo eso se puede mover al create del serializador para que el serializador se pueda usar directamente desde el post y que todo quede más limpio. Le voy a dedicar una hora más o menos para ver si avanzo. También me gustaría hacer que una vez que al registrar se oculte el formulario y que aparezca un mensaje de si funcionó o no y se redirija al login para que se loguee. Y creo que voy a ver cómo sería la lógica del login para mantenerse logueado.
